kafka-ui-react-app/src/components/Version/Version.tsx

```diff
@@ -1,20 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { gitCommitPath } from 'lib/paths';
+import { GIT_REPO_LATEST_RELEASE_LINK } from 'lib/constants';
+
+import compareVersions from './compareVersions';
 
 export interface VesionProps {
   tag?: string;
   commit?: string;
 }
 
 const Version: React.FC<VesionProps> = ({ tag, commit }) => {
+  const [latestTag, setLatestTag] = useState<string>('');
+  useEffect(() => {
+    if (tag) {
+      fetch(GIT_REPO_LATEST_RELEASE_LINK)
+        .then((response) => response.json())
+        .then((data) => {
+          setLatestTag(data.tag_name);
+        });
```

Comment:
We should be ready to case when customer runs our App internally without internet.